src/EventStream.js

```diff
@@ -156,16 +159,21 @@ class EventStream extends EventEmitter {
 				if (this.data.length > 0 && this.event) {
 					const event = JSON.parse(this.data);
 					event.name = this.eventName || '';
-					if (this.eventName !== 'event') {
-						this.emit(this.eventName, event);
+					try {
+						if (this.eventName !== 'event') {
+							this.emit(this.eventName, event);
+						}
+						this.emit('event', event);
```

Comment:
i realize this isn't a change you introduced but i'm wondering why the `if` statement here is needed at all?

e.g. simply:

`this.emit(this.eventName, event);`

Reply:
Because this would emit the Node event twice if the Particle event was name `event`. It's a bad pattern to emit a Node event from the Particle event name. The Particle event `error` also clashes with the Node error event.